test/unit/test_util.c

```diff
@@ -113,6 +113,42 @@ bool test_autonames(void) {
 	mu_end;
 }
 
+bool test_file_slurp(void) {
+
+#ifdef __WINDOWS__
+#define S_IRWXU _S_IREAD | _S_IWRITE
+#endif
+
+	char* test_file =  "./empty_file";
+	size_t s;
+	char* content;
+	char* some_words = "some words";
+
+	close(open(test_file, O_CREAT, S_IRWXU));
+
+	content = r_file_slurp(test_file, &s);
```

Comment:
defnie and assign the variable in the same line and add a space bfore (

Reply:
done